themes/default/content/docs/intro/concepts/resources/_index.md

```diff
@@ -28,13 +28,13 @@ menu:
         "#version": "/docs/intro/concepts/resources/options/version",
         "#components": "/docs/intro/concepts/resources/components",
         "#authoring-a-new-component-resource": "/docs/intro/concepts/resources/components/#authoring-a-new-component-resource",
-        "#creating-a-child-resource": "/docs/intro/concepts/resources/components/#creating-a-child-resource",
+        "#creating-child-resources": "/docs/intro/concepts/resources/components/#creating-child-resources",
         "#registering-component-outputs": "/docs/intro/concepts/resources/components/#registering-component-outputs",
         "#inheriting-resource-providers": "/docs/intro/concepts/resources/components/#inheriting-resource-providers",
         "#providers": "/docs/intro/concepts/resources/providers",
         "#default-provider-configuration": "/docs/intro/concepts/resources/providers/#default-provider-configuration",
         "#explicit-provider-configuration": "/docs/intro/concepts/resources/providers/#explicit-provider-configuration",
-        "#dynamicproviders": "/docs/intro/concepts/resources/dynamicproviders",
+        "#dynamicproviders": "/docs/intro/concepts/resources/dynamic-providers",
```

Comment:
Note that for this heading, [we applied a custom anchor](https://raw.githubusercontent.com/pulumi/pulumi-hugo/12c547508213699a93cde3aae460f610e21d6a24/themes/default/content/docs/intro/concepts/resources.md) (`dynamicproviders`) -- I'm not sure why. 🤷